DrawerKit/DrawerKit/Internal API/GeometryEvaluator.swift

```diff
@@ -89,11 +94,12 @@ extension GeometryEvaluator {
 
     static func drawerPositionY(for state: DrawerState,
                                 drawerPartialHeight: CGFloat,
+                                drawerCollapsedHeight: CGFloat,
                                 containerViewHeight: CGFloat,
                                 drawerFullY: CGFloat) -> CGFloat {
         switch state {
         case .collapsed:
-            return containerViewHeight
+            return containerViewHeight - drawerCollapsedHeight
```

Comment:
If I understand correctly, I think the subtraction here should only happen when `canDismissCollapsedDrawer` is `false`. If it's `true`, then the previous behaviour (ie, no subtraction) should be the one that takes effect. In other words, I think this should be

```swift
let offset: CGFloat = canDismissCollapsedDrawer ? 0 : drawerCollapsedHeight
return containerViewHeight - offset
```

Please make sure to test the behaviour for *both* values of `canDismissCollapsedDrawer` and make sure that the behaviour for `canDismissCollapsedDrawer == true` matches the current behaviour.